Makefile

```diff
@@ -309,6 +309,15 @@ html_report: ## Generate HTML  report out of the last ran integration test logs.
 	@echo "-----------------------------------------------------------------------------------"
 	@open $(CURDIR)/out/testout_$(COMMIT_SHORT).html || true
 
+.PHONY: html_report_windows
+html_report_windows: ## Generate the HTML Report out of the last ran integration test logs on Windows.
```

Comment:
can we make this automatic ?
like if we call make html_report and it is in windows, we automatically call this target instead ?

Reply:
Done. Works now on both Linux and Windows and uses the `GOOS` environment variable.